chia/_tests/blockchain/test_blockchain.py

```diff
@@ -1890,7 +1890,7 @@ async def test_conditions(
 
         wt: WalletTool = bt.get_pool_wallet_tool()
```

Comment:
`WalletTool` is a *test* facility for the full node, right? Or perhaps part of the simulator (which I think has to be part for the full node for tests). Alternatively it could be its own project that depends both on the full node and the wallet. But then the full node *tests* would have to also depend on it.